ivadomed/config/example_loader_v2_configs.py

```diff
@@ -0,0 +1,398 @@
+from __future__ import annotations
```

Comment:
Optional in Python 3.7 to 3.10 ref: https://docs.python.org/3.11/library/__future__.html